invenio_records_resources/services/files/service.py

```diff
@@ -190,7 +190,7 @@ def delete_file(self, identity, id_, file_key, uow=None):
         :raises FileKeyNotFoundError: If the record has no file for the ``file_key``
         """
         record = self._get_record(id_, identity, "delete_files", file_key=file_key)
-        deleted_file = record.files.delete(file_key)
+        deleted_file = record.files.delete(file_key, remove_rf=True)
```

Comment:
not connected with the content of this PR, but to fix in this repo:

typo in the base class of the file service component
https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/files/components/base.py#L44